### ISSUE TITLE
Driver build fails with kernel 5.15.xx

### ISSUE THREAD

**Issue by f-laurens** [pguyot/cr14#6]
```
/usr/bin/git pull -v
From https://github.com/pguyot/cr14
 = [up to date]      release    -> origin/release
 = [up to date]      master     -> origin/master
Already up to date.
Building cr14 driver for kernel 5.15.30+.
/usr/bin/make KERNELRELEASE=5.15.30+ clean && /usr/bin/make KERNELRELEASE=5.15.30+ && /usr/bin/sudo /usr/bin/make KERNELRELEASE=5.15.30+ install && /usr/bin/make KERNELRELEASE=5.15.30+ clean
make -C /lib/modules/5.15.30+/build M=/home/pi/cr14 clean
make[1]: Entering directory '/usr/src/linux-headers-5.15.30+'
make[1]: Leaving directory '/usr/src/linux-headers-5.15.30+'
make -C /lib/modules/5.15.30+/build M=/home/pi/cr14 modules
make[1]: Entering directory '/usr/src/linux-headers-5.15.30+'
  CC [M]  /home/pi/cr14/cr14.o
  MODPOST /home/pi/cr14/Module.symvers
  CC [M]  /home/pi/cr14/cr14.mod.o
  LD [M]  /home/pi/cr14/cr14.ko
make[1]: Leaving directory '/usr/src/linux-headers-5.15.30+'
make: *** No rule to make target 'cr14.dtbo', needed by 'install'.  Stop.
```

**Comment by Commit-La-Grenouille** [comment]
Change validated and raised for review : https://github.com/pguyot/cr14/pull/7

**Comment by pguyot** [comment]
Fixed with #7
